tests/annalyns-infiltration/exemplar-solution/expected_analysis.json

```diff
@@ -0,0 +1,11 @@
+{
+  "comments": [
+    {
+      "comment": "java.general.exemplar",
+      "params": {
+        "exerciseName": "Annalyn\u0027s Infiltration"
```

Comment:
Any idea why the apostrophe is printed as a unicode character here? I'm not sure whether this will render correctly when presented on the website. Maybe @ErikSchierboom can shine some light on whether this is supported.

Reply:
I think it will be fine.